### PR TITLE
Minor fixes and readme

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello, 

I have updated Readme.md and fixed toast toggle issue.

Live README.md link : https://github.com/ApurvaSawant11/Dazzle-UI-Components/blob/minor-fixes-and-readme/README.md

Please review and suggest

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
Great you can merge it

**Review by keshavkumar-in — APPROVED** [review]
Good to merge, everything looks fine to me.

File: script.js [hunk 1]
```diff
@@ -38,18 +38,18 @@ const iconToast = document.querySelector(".icon-toast-container");
 const iconToastClose = document.querySelector('.icon-toast-close');
```
Comment (Developers-codz) [inline script.js:38]: Using id selector will be better than class selector in Js

**Review by Developers-codz — APPROVED** [review]
Everything else look good to merge

**Review by Anurag021 — APPROVED** [review]
Code looks good to me.

File: script.js (hunk 1 above)
Comment (ApurvaSawant11) [reply script.js:38]: Okay.  Will keep that in mind! Thanks

**Pull request merged**